### PR TITLE
More responsive Library layout + Togglable search section at low resolutions

### PR DESCRIPTION
**Pull request by ELowry**
This modification brings 2 main things:

1. The library is now more responsive, with a number of columns that changes dynamically based on the window width (in px). This is based on css `@media` queries set to follow the same intervals as the default Stadia page's css (+ some more intervals based on the ones that were already being used in this plugin's code).
2. The main "search" panel at the left of the library is now masked by default on lower resolutions (`<1024px`) with a button to reveal it placed to the left of the "Auto Update" button.

The system described in point 2 has **not yet been tested** and **may need some refining** (specifically, the icon may need to be changed to be more obvious, and a tooltip may even be necessary).

### CODE REVIEW

File: src/components/LibraryFilter.ts [hunk 1]
```diff
@@ -445,6 +445,32 @@ export class LibraryFilter extends Component {
                             .child(visibleDropdown)
                     )
             )
+            .child(
```
Comment (Mafrans) [inline src/components/LibraryFilter.ts:488]: What happened here?

Comment (ELowry) [reply src/components/LibraryFilter.ts:488]: This is where I added the "searchcolumn-toggle" button+icon, which is used to show/hide the search "tab" when on lower resolutions. It sits to the left of the "Auto Update" button and is only visible on lower resolutions (<1024px).

**Comment by ELowry** [comment]
The `searchcolumn-toggle` icon should now "shift" from a `search` icon when the panel is hidden to an `arrow_back` icon when it is displayed.

**Comment by Mafrans** [comment]
I'm getting this, maybe some more testing is necessary?

**Comment by ELowry** [comment]
Should be fixed, with a few enhancements to the animations/display =)

**Comment by ELowry** [comment]
Replaced by #164.

**Pull request closed without merging**